### ISSUE TITLE
5000IVs for WEP Cracking isn't enough

### ISSUE THREAD

**Issue by drifter666** [v1s1t0r1sh3r3/airgeddon#198]
#### What is your airgeddon version?

Version 8.12

#### What is your Linux O.S. and which version?

Kali Linux 2018.4

#### Which is the chipset of your wireless card?

Qualcomm Atheros AR9271 (TP-Link TL-WN722N V1)

#### Describe the issue and the steps to reproduce it

Aircrack-ng automatically starts after collecting 5000 IVs but it isn't enough and says to try again at 10000 IVs. I was thinking that it should auto try again when 10000 IVs is collected but it doesn't. I changed the airgeddon script to start at 10000 IVs and it successfully started and cracked it fine. 

So is it suppose to auto crack again when 10000IVs was reached?

**Comment by v1s1t0r1sh3r3** [comment]
Oh my god... it seems you never worked with WEP and aircrack. airgeddon is working with aircrack for this purpuse, so it works in the same way as aircrack works. So yes, it starts to try to decrypt when 5000 Data (IVs) are captured. Of course, 5000 IVs are not enough... but you must keep capturing. Once 10000 Data are captured it starts again to try to decrypt. Of course 10000 Datas are not enough IVs to decrypt the network and then it waits to have 5000 IVs more, so at 15000 IVs it starts again, etc etc... I tries each 5000 IVs. A standard network can start to be decrypted from 35000 to 100000 IVs. You must be patient, it depends of a lot of factors even if you are lucky you can get it very soon, but that is something out of our hands.

This behavior is nice and is the way on which aircrack works, and airgeddon uses aircrack so is a behavior not developed by airgeddon, is designed in aircrack so we can't do anything. But as I said, I think is really nice.

Do a lot of tests, try different cards, over different WEP access points or you can try even different aircrack versions, you'll see what I'm talking about, and it is working flawlessly. If for any strange reason this is not working for you, it must not be related to airgeddon. That's all I can tell you.

**Comment by drifter666** [comment]
It doesn't retry at 10000 IVs. That's what I was saying. It just stops. I collected 30000IVs and the screen just stays at 5000IVs, Please try again at 10000 IVs.

Instead of being ignorant, try asking more questions.

**Comment by v1s1t0r1sh3r3** [comment]
I don't know what can be happening in your case because once the capture start, is aircrack in charge to count the IVs and it does the new cracking attempt automatically. I did this morning a test and I wasn't able to reproduce your problem. Put more data.... what is your access point? what is your aircrack version?

I can suggest to you to try it in other environment... maybe a different virtual machine over a different AP. Are you having the same strange behavior?

**Comment by drifter666** [comment]
I'm definitely leaning towards Kali as the culprit. I've rebuilt a few VMs and I'm getting odd behaviours with aircrack directly now. I've tried different hardware and same thing. With the rolling releases, I hit something weird once in awhile. 

Sorry to have opened the ticket. I found this project and I'm really liking it. When I saw this issue, I just didn't want something silly to be broken without notifying you since I experienced it on two different machines. One a VM and one a laptop install.
